config/task.yaml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2022 Chainguard, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: apko
+spec:
+  description: "An apko task for building images."
+  params:
+    - name: dev.mink.sources.bundle
+      description: A self-extracting container image of source
+    - name: dev.mink.images.target
+      description: Where to publish an image.
+    - name: path
+      description: The bundle-relative path to the config file to build.
+      default: ./config.yaml
+
+  results:
+    - name: IMAGES
+      description: The list of images produced by this task (for chains)
+    - name: dev.mink.images.digest
+      description: The digest of the resulting image.
+
+  steps:
+    - name: extract-bundle
+      image: $(params["dev.mink.sources.bundle"])
+      workingDir: /workspace
+
+    - name: build-image
+      image: ko://chainguard.dev/apko/cmd/apko
+      workingDir: /workspace
+      args:
+      - build
```

Comment:
Are there plans to have apko push by itself?

Reply:
Possibly, but the beauty of the task is that we can just change it to use that once it can 😉 

... or perhaps we allow the `apko` step to run hermetically (it would likely need some sort of prefetch step) and keep things this way.

Reply:
Yes, `apko publish` is coming next.

Reply:
That's gonna be sweet.  We should add in all of @imjasonh cool universal cred helper stuff 🤩